docs/guide/security/policy.adoc

```diff
@@ -19,7 +19,7 @@ camelctx.addRoutes(new RouteBuilder() {
     }
 });
 
-This does actually no do any authentication/authorization as part of the Camel message processing. Instead it associates the credentials that come with the Camel Exchange with the call into the EJB3 layer.
+This does not do any authentication and authorization, as a  part of the camel message processing. Instead, it associates the credentials that come with the Camel Exchange with the call into the EJB3 layer.
```

Comment:
Unnecessary additional space in 'as a  part'.

Reply:
Fixed, as suggested.